scripts/initialize-build-host.sh

```diff
@@ -433,7 +433,7 @@ then
     # then
     #     :
     # else
-        echo "build-artifacts-cache.cloud.cfengine.com ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC6qcxCQgtubv9WEhrAyMEFFMLLEjirk0p0Ru+vATioEIyw7gBFfOWOp/dBfsF6fuiY1vt3IsBx4u1DkS4j8x7DjB8X2dIcBia2jt2D3sBdDFb/nc7ZnWfFf/E7dWoiF0WKvxZ62RwjyZuyz9TmL1d3jlIyuRimkhgwnuRAMyymJ5YbxvvfTH01OuGS/0pkqkLAxomRyJTv6qcGr1rOPd5FuySwOO5M/tGkajJppKC+8u/RCyWfgu1khrBmi6PevXTaoJ/lQyexexZK0HVsA5G1U/+ipO18DqaCCAnHvZ/AKt+yYmoe9RtLfx0T7DHinEV1yj4ynUj7EqudCrLOorg5 root@yoctobuild-sstate-cache"  > $KNOWN_HOSTS_FILE
+        echo "|1|D9BiCANqKBjcdQIy8QZMnmmzboY=|u2XzAOAbHDWyrZLC3aiX16VxUS0= ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAINMJKl282VQSz4EMMypJjATu21A9SxQA1XoTslIOID16" > $KNOWN_HOSTS_FILE
```

Comment:
this did not help :( will try something else. Maybe something about ubuntu 22 doesn't like this key? The build-artifacts-cache.cloud.cfengine.com host returns an ed25519 host key instead of rsa and that's the symptom I have currently during builds.